### PR TITLE
feat: added history page

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added History Page**
- Added API calls for history.
- Added remove from history button.
- Added a clear history button.
- History will get updated when the user clicks on any video to watch it.

### CODE REVIEW

**Review by sharathnair9999 — APPROVED** [review]
Features look good.
UI improvements: Single Video page has some horizontal scroll issue
add _redirects file in ur public folder. deployed version wont reload ortherwise

File: src/utilities/History/add-history.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+export const addToHistory = async (video, dispatchVideo) => {
+    const token = localStorage.getItem("token");
```
Comment (parthgabani777) [inline src/utilities/History/add-history.js:4]: consider edge cases if a user is not signed in when adding to history.

**Review by parthgabani777 — APPROVED** [review]
Code looks well structured 
Good use of the separate file for API calls.

File: src/utilities/History/add-history.js (hunk 1 above)
Comment (nerdyspook) [reply src/utilities/History/add-history.js:4]: Thanks for the suggestion, will do it while refactoring it later.

**Pull request merged**